### PR TITLE
HDDS-9483. ozone freon --server is broken by HDDS-6176.

### PR DESCRIPTION
**Pull request by jojochuang**
## What changes were proposed in this pull request?
Specify a default temporary directory for HttpServer2 for 'ozone freon --server' command.

Please describe your PR in detail:
Essentially copy-pasted from HDDS-8253

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-9483

## How was this patch tested?

Manually tested on a cluster.

### CODE REVIEW

**Comment by adoroszlai** [comment]
Thanks @jojochuang for the patch.

> Essentially copy-pasted from [HDDS-8253](https://issues.apache.org/jira/browse/HDDS-8253)

Instead of copying, can you please extract to some utility class (e.g. `HddsServerUtil`)?

File: hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/Freon.java [hunk 1]
```diff
@@ -111,9 +118,31 @@ public void stopHttpServer() {
     }
   }
 
+  private void setHttpBaseDir()
```
Comment (swamirishi) [inline hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/Freon.java:121]: this method can be set as a util function in ozone-common module.

**Comment by kerneltime** [comment]
You can move the code aside or alternatively create in the current working directory for the CLI and avoid having these configs defined for the CLI. It is perfectly fine to create artifacts in the CWD for a binary and cleaning it up at the end.

**Review by adoroszlai — APPROVED** [review]
Thanks @jojochuang for updating the patch.

**Comment by adoroszlai** [comment]
Thanks @jojochuang for the fix, @kerneltime, @swamirishi for the review.

**Comment by jojochuang** [comment]
Thanks Attila Ritesh and Swami!

**Pull request merged**